api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -211,3 +212,98 @@ def create_offers_for_each_subcategory() -> None:
                 offer__subcategoryId=subcategory.id,
                 quantity=random.randint(10, 100),
             )
+
+
+def create_offers_with_same_author() -> None:
+    # an author with 12 different books
+    author = Fake.name()
+    venues = [
+        offerers_factories.VenueFactory(
+            name="same author " + str(venue["name"]),
+            venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+            isPermanent=True,
+            latitude=venue["latitude"],
+            longitude=venue["longitude"],
+            address=venue["address"],
+            postalCode=venue["postalCode"],
+            city=venue["city"],
+            departementCode=venue["departementCode"],
+        )
+        for venue in random.choices(venues_mock.venues, k=4)
+    ]
+    for venue in venues:
+        for _ in range(4):
+            create_offer_with_ean(Fake.ean13(), venue, author=author)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏On a 4*4=16 offres ici, le commentaire ment
```python
>>> for i in range(4):
...   print(i)
... 
0
1
2
3
```

Reply:
effectivement il faut que je réapprenne mes tableaux de multiplication